server/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
This seems like a compiled file. Could u put the whole /bin folder into .gitignore

Reply:
I think the `bin/www` file is required to start the app [Express 4 onwards](https://stackoverflow.com/a/23249547/8316412). I've removed it and modified the `app.js` with its contents to start the way it was Express 3 and before.

Reply:
Hey @RonLek Sorry! I wasn't aware of this :see_no_evil: